### PR TITLE
Add netid support for swipes

### PR DESCRIPTION
**Pull request by JParisFerrer**
Also adds editing netid on the student page for testing

### CODE REVIEW

File: src/pages/sectionMeeting/sectionMeeting.js [hunk 1]
```diff
@@ -37,8 +48,34 @@ router.post(
...
+      if (uin.length > 0 && Number.isNaN(parseInt(uin[0],10))) {
+        const params = {
+          netid: uin,
+          ciTerm: req.body.ciTerm,
+          ciName: req.body.ciName,
+          ciYear: Number.parseInt(req.body.ciYear, 10),
+        };
+        
+        const results = await dbDriver.asyncQuery(sql.find_matching_student, params);
+        if (results.rows.length > 0) {
```
Comment (Binary-Eater) [inline src/pages/sectionMeeting/sectionMeeting.js:67]: @JParisFerrer we don't handle the case when no matching student is found and `uin` is a netid and not a valid uin, do we?

Comment (Binary-Eater) [reply src/pages/sectionMeeting/sectionMeeting.js:67]: If you don't account for it, you can just use `ERR` to show that an invalid `netId` was entered.

Comment (Binary-Eater) [reply src/pages/sectionMeeting/sectionMeeting.js:67]: https://github.com/nwalters512/attendance/blob/200f535313ff25596cab71bf9024da78df6f05ea/src/pages/home/home.js#L28

Comment (JParisFerrer) [reply src/pages/sectionMeeting/sectionMeeting.js:67]: This should be handled by L85 (and seems to be for me)

Comment (JParisFerrer) [reply src/pages/sectionMeeting/sectionMeeting.js:67]: Could change for code clarity though

Comment (JParisFerrer) [reply src/pages/sectionMeeting/sectionMeeting.js:67]: done

**Review by Binary-Eater — APPROVED** [review]
Tested it out and does not seem to cause any breaking changes.

**Review by Binary-Eater — COMMENTED** [review]
May want to use this instead of hardcoding the pg error codes.

https://www.npmjs.com/package/pg-error-constants

**Pull request merged**